.golangci.toml

```diff
@@ -183,6 +179,11 @@ text = "501"
 linters = ["gosec"]
 text = "404"
 
+# https://stackoverflow.com/a/68247837/767660
+[[issues.exclude-rules]]
+linters = ["gosec"]
+text = "G601"
+
```

Comment:
Please check https://stackoverflow.com/a/68247837/767660
I think it is ok to exclude it.

Reply:
I think so long as Grafana requires < Go 1.22.x, it's not safe to exclude, since implicit memory aliasing in for loops is still a problem.